### ISSUE TITLE
Deploy function without starting it

### ISSUE THREAD

**Issue by Guchman** [apache/pulsar#5366]
I want to start and stop function on different system events.
Example:
I have dataflows A and B.

"A" is initial db state, "B" is updates from journal log.
I need datastream where B goes strongly after A.
So I have Bridge function that will shot circuit B on A

I could deploy function when event occurred, but I don't want my application service to deploy functions. It must simply start and stop it.

**Comment by aahmed-se** [comment]
Functions are not meant to start stop on events, they respond to messages on  topic messages, they are stateful be design. They are closer to stream processing.

**Comment by Guchman** [comment]
@aahmed-se I mean external control of function, stop and start that with API.
Anyway, it's may be useful to deploy full stream pipeline before starting that

Actually I have stream processing pipeline, but complex one, so I need to merge two streams sequentially.

**Comment by tisonkun** [comment]
Closed as stale. Please open a new issue if it's still relevant to the maintained versions.
